lib/Employee.js

```diff
@@ -0,0 +1,23 @@
+class Employee {
+  constructor({ name, id, email }) {
+    this.name = name;
+    this.id = id;
+    this.email = email;
+  }
+
+  getName() {
+    return this.name;
+  }
+
+  getId() {
+    return this.id;
+  }
+
+  getEmail() {
+    return this.email;
+  }
+
+  getRole() {}
```

Comment:
This can be removed as you do not need to declare the getRole method here - you are declaring this when this class is extended.